### PR TITLE
[VEN-1359] Add announcement about SXP disabling

### PR DESCRIPTION
**Pull request by therealemjy**
## Jira ticket(s)

VEN-1359

## Changes

- add announcement about SXP disabling (currently present on `master`)

### CODE REVIEW

File: src/components/Announcement/index.tsx [hunk 1]
```diff
@@ -14,7 +14,17 @@ export interface AnnouncementProps {
 
 export const Announcement: React.FC<AnnouncementProps> = ({ token }) => {
   const styles = useStyles();
-  const { Trans } = useTranslation();
+  const { Trans, t } = useTranslation();
+
+  // TRX disabling
```
Comment (gleiser-oliveira) [inline src/components/Announcement/index.tsx]:
```suggestion
  // SPX disabling
```

Comment (therealemjy) [reply src/components/Announcement/index.tsx]: Good catch!

File: src/constants/tokens/swap/mainnetPancakeSwapTokens.ts
Comment (gleiser-oliveira) [inline src/constants/tokens/swap/mainnetPancakeSwapTokens.ts:1]: @therealemjy I think this might have gotten picked up by accident

Comment (therealemjy) [reply src/constants/tokens/swap/mainnetPancakeSwapTokens.ts:1]: Indeed, sorry for the confusion I forgot to rebase with `develop`.

**Review by gleiser-oliveira — APPROVED** [review]

**Pull request merged**